gradle/dependencies.gradle

```diff
@@ -112,6 +112,7 @@ versions += [
   spotbugs: "3.1.12",
   spotbugsPlugin: "3.0.0",
   spotlessPlugin: "3.27.1",
+  testRetryPlugin: "1.0.2",
```

Comment:
How about using latest version (1.1.0)?

Reply:
I also noticed it was released shortly after this PR was submitted. Updated.